FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -296,13 +296,13 @@ - (void)handleOauthDiscoveryCompletion:(OIDServiceConfiguration *_Nullable)confi
           NSError *signInError = nil;
           if (error.code == OIDErrorCodeUserCanceledAuthorizationFlow) {
             // User cancelled auth flow
-            NSLog(@"Tester cancelled sign in flow");
+            FIRFADInfoLog(@"Tester cancelled sign in flow");
             signInError =
                 [self NSErrorForErrorCodeAndMessage:FIRAppDistributionErrorAuthenticationCancelled
                                             message:kAuthCancelledErrorMessage];
           } else {
             // Error in the auth flow
-            NSLog(@"Tester sign in error - %@", [error localizedDescription]);
+            FIRFADInfoLog(@"Tester sign in error - %@", [error localizedDescription]);
```

Comment:
log level here too - this is likely my own mistakes coming back to get me.

Reply:
Done!